apps/opik-backend/config.yml

```diff
@@ -80,3 +80,6 @@ usageReport:
 
 metadata:
   version: ${OPIK_VERSION:-latest}
+
+cors:
+  enabled: ${CORS:-false}
```

Comment:
Minor
```suggestion
  enabled: ${OPIK_CORS:-false}
```

Reply:
IMO, the best suggestion here, which also follows the existing convention, would be `CORS_ENABLED`.